tests/test_sevm.py

```diff
@@ -68,7 +99,9 @@ def test_run(hexcode, stack, pc, opcode: int, sevm, solver, storage):
     assert ex.pc == pc
     assert ex.current_opcode() == int_of(opcode)
 
+
 def byte_of(i, x):
+    # fmt: off
```

Comment:
this is the only customization I made

Reply:
let's not make an exception, as it is subjective to determine when it is acceptable.

there are other places where the formatted code is very ugly.  we can work on them in separate prs by rewriting them in a formatting-friendly way.

Reply:
ah, i will take back. this is a test file, and i couldn't think of a good way to write it in the format friendly way (it needs to be verbose to cross-check with the main code logic.)

Reply:
<img width="735" alt="image" src="https://github.com/a16z/halmos/assets/85039585/96c4b1a7-c68d-4401-a3bb-f195a31e4c28">

Reply:
yeah I was going to say I agree about avoiding exceptions, but given that this is a test file and the extreme nesting makes black spiral into madness, so I think an exception is fine 

<img width="735" alt="image" src="https://github.com/a16z/halmos/assets/85039585/74153b1f-abf8-4659-8138-5abb7489d206">